### PR TITLE
Update get-missing-kbs-machine.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated permissions required for the API

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6051

File: microsoft-365/security/defender-endpoint/get-missing-kbs-machine.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ Retrieves missing KBs (security updates) by device ID
 ```http
 GET /api/machines/{machineId}/getmissingkbs
 ```
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Use Microsoft Defender for Endpoint APIs](apis-intro.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-missing-kbs-machine.md]:
```suggestion
The following permission is required to call this API. To learn more, including how to choose permissions, see [Use Microsoft Defender for Endpoint APIs](apis-intro.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**